README.md

```diff
@@ -103,6 +103,32 @@ terminus(server, options);
 server.listen(PORT || 3000);
 ```
 
+## How to set Terminus up with Kubernetes?
+
+When a pod is deleted, Kubernetes will notify it and will wait for `gracePeriod` seconds before killing it.
+
+During that time window (30 seconds by default), the pod is in the `terminating` state and will be removed from any Services by a controller. The pod itself needs to catch the `SIGTERM` signal, and start failing any readiness probes.
+
+During this time, it is possible that load-balancers don't remove the pods "in time", and when the pod dies, it kills live connections.
```

Comment:
Is this specific to the nginx ingress controller?  Or do other implementations also have this result?

Reply:
afaik, this is how all implementations work

Reply:
any chance @peterkuiper or @thockin that you can verify that?

Reply:
I think its relevant because the service itself can be a single IP that fronts the pods and it won't be taken out of the DNS pool until readiness check tells it.

Reply:
the nginx ingress controller routes directly to Pods (not via the Service).  If it routed via the Service, we wouldn't have this issue (https://kubernetes.io/docs/concepts/workloads/pods/pod-lifecycle/)

Reply:
Also relevant:

https://github.com/kubernetes/contrib/issues/1140
https://github.com/godaddy/terminus/issues/31